package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "search-by-time-github",
+	"version": "1.0.0",
+	"description": "<!-- PROJECT SHIELDS -->",
+	"main": "index.js",
+	"directories": {
+		"doc": "docs"
+	},
+	"scripts": {
+		"lint": "eslint --ext .js src",
+		"lint:fix": "eslint src --ext .js --fix",
+		"format": "prettier --write \"**/*.+(js|html|css|md)\""
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Azordev/search-by-time-github.git"
```

Comment:
this `git+` url is ok? 

I mean, it really start for: "git+"?

Reply:
yes, that is the correct format I think